ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/CatIndexTool.java

```diff
@@ -106,13 +119,13 @@ public <T> void run(Map<String, String> parameters, ActionListener<T> listener)
         final String[] indices = indexList.toArray(Strings.EMPTY_ARRAY);
 
         final IndicesOptions indicesOptions = IndicesOptions.strictExpand();
-        final boolean local = parameters.containsKey("local") ? Boolean.parseBoolean("local") : false;
-        final TimeValue clusterManagerNodeTimeout = DEFAULT_CLUSTER_MANAGER_NODE_TIMEOUT;
+        final boolean local = parameters.containsKey("local") && Boolean.parseBoolean(parameters.get("local"));
```

Comment:
Thanks for fixing what was obviously a bug in the previous version. :)
```suggestion
        final boolean local = Boolean.parseBoolean(parameters.get("local"));
```
`parseBoolean(null)` returns false so you don't need to bother with checking the key existence.